katharsis-client/src/main/java/io/katharsis/client/KatharsisClient.java

```diff
@@ -166,6 +166,21 @@ public RelationshipRepository buildRepository() {
 		ResourceRepositoryAdapter repositoryAdapter = entry.getResourceRepository(null);
 		return (ResourceRepositoryStub<T, ID>) repositoryAdapter.getResourceRepository();
 	}
+	
+	/**
+	 * @param resourceClass resource class
+	 * @return stub for the given resourceClass
+	 */
+	@SuppressWarnings({ "unchecked", "rawtypes" })
+	public <T, ID extends Serializable> QuerySpecResourceRepositoryStub<T, ID> getQuerySpecRepository(Class<T> resourceClass) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Rename this generic name to match the regular expression '^[A-Z][0-9]?$'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS00119)